### PR TITLE
@orta => specify target for artwork button actions

### LINKED ISSUE #474: artwork buttons do not work
go to artwork page, try to tap contact button, nothing happens, tapping heart, share, auction results, nothing happens. Go to different artwork try all the same buttons again, nothing. 

http://cl.ly/image/0q210b0z1j0N

### PR DESCRIPTION
**Pull request by 1aurabrown**
Fixes #474.

### CODE REVIEW

**Comment by 1aurabrown** [comment]
@ashfurrow suggested that i set up delegates for the various buttons contained in the artwork view and ensure that there is a target specified for the button actions. I guess this could be structured a few different ways, but this made sense to me.

**Comment by orta** [comment]
Ah, such a shame, but `simple + works` > `elegant`

**Comment by ashfurrow** [comment]
Tests are passing locally. 

@orta ready for review.

File: Artsy Tests/ARArtworkActionsViewTests.m [hunk 1]
```diff
@@ -267,4 +277,104 @@ - (void)setupCountdownView;
     });
 });
 
+describe(@"mocked artwork promises", ^{
+    beforeEach(^{
+        id artwork = [OCMockObject mockForClass:[Artwork class]];
```
Comment (orta) [inline Artsy Tests/ARArtworkActionsViewTests.m]: this is an aside, but OCMock 3.0 is backwards compatible, and has a better Specta-Like DSL

Comment (ashfurrow) [inline Artsy Tests/ARArtworkActionsViewTests.m]: Mmm. I'll start using the new syntax.

File: Artsy/Classes/View Controllers/ARArtworkViewController+ButtonActions.m [hunk 2]
```diff
@@ -76,7 +80,33 @@ - (void)tappedArtworkViewInMap:(id)sender
     }];
 }
 
-- (void)tappedBidButton:(ARBidButton *)sender
+#pragma mark - ARArtworkActionsViewButtonDelegate
+
+- (void)tappedContactGallery
+{
```
Comment (orta) [inline Artsy/Classes/View Controllers/ARArtworkViewController+ButtonActions.m]: I wonder if some of these should go into the switchboard

File: Gemfile.lock [hunk 3]
```diff
@@ -243,6 +243,3 @@ DEPENDENCIES
   second_curtain
   shenzhen
   xcpretty
-
-BUNDLED WITH
-   1.10.0
```
Comment (orta) [inline Gemfile.lock]: `gem install bundler`

**Comment by orta** [comment]
Alright, can we also get a CHANGELOG entry too please

File: Gemfile.lock (hunk 3 above)
Comment (ashfurrow) [inline Gemfile.lock]: Ah, yeah. @1aurabrown you too.

**Comment by ashfurrow** [comment]
:green_apple:

**Comment by orta** [comment]
Good show you two :+1:

**Pull request merged**